src/v2/views/MonsterCollectionOverlay/Migration.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useMemo, useReducer } from "react";
+import React, { useEffect, useMemo, useReducer } from "react";
```

Comment:
I'm not sure but I can't find where the `useEffect` was used 🤔

Reply:
Ooh yeah I was doing some experiments while I was working on this 😂 It is a unused import.